.travis.yml

```diff
@@ -1,59 +1,59 @@
-language: cpp 
```

Comment:
This file (`.travis.yml`) and `.travis/package.txt` are generated with scripts. If the trailing whitespace is removed, the script should be updated. Otherwise, if you rerun the script, the trailing whitespace is back and you get the inverse diff.